include/envoy/http/codec.h

```diff
@@ -36,6 +36,11 @@ const char MaxResponseHeadersCountOverrideKey[] =
 
 class Stream;
 
+enum class GoAwayErrorCode {
```

Comment:
sorry should have mention this but this should have a doc comment as its in `include/`